### ISSUE TITLE
.storekit path incorrect in the .xcscheme file

### ISSUE THREAD

**Issue by anandios** [yonaskolb/XcodeGen#1281]
Hello,

While we set the `storeKitConfiguration` parameter to be something like `storeKitConfiguration: Configuration/App-Development.storekit` in the project.yml file and then generate the project file, in the respective scheme, the storekit path is not being set correctly by the Xcodegen. The path that it adds in the scheme is `../../Configuration/App-Development.storekit`, while the real path that actually works is `../Configuration/App-Development.storekit`.

Can someone please look into this?
Thank you.
Anand

**Comment by yonaskolb** [comment]
You can control the prefix with `options.schemePathPrefix` https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#options

**Comment by TemperMichael** [comment]
Hello, I have a similar issue that the StoreKit config just does not get set in the scheme, I already tried all kind of variations like with & without schemePathPrefix, with ../ and ../../ and even run XcodeGen locally to print out what get's set internally and it seems to be set correctly. I even compared the configs path with the value that is shown in source control when I set the configuration manually in Xcode; and even though the path's match correctly, it still does not get set. Am I doing something wrong or could it be that this is currently just not working? I set the scheme in my project.yml file like this:

```
schemes:
    MTExamples - StoreKit Tests:
        build:
            targets:
                MTExamples: all
            parallelizeBuild: true
            buildImplicitDependencies: true
        run:
            storeKitConfiguration: "MTExamples/SupportingFiles/StoreKitTestConfiguration/Configuration.storekit"
```

I am just running a bit out of ideas - thanks in advance!
